### ISSUE TITLE
Rename repo

### ISSUE THREAD

**Issue by omginbd** [dansteren/mlt-ts#1]
Though it's written in typescript it can be used in plain old node land. It might be more descriptive if the repo / npm package was named mlt-node. Nothing, critical, but it might make it easier for students to find in the future!

**Comment by dansteren** [comment]
Good call. I was so caught up in using typescript that I didn't realize until I had created it. I'm a little hesitant to change it right now since its already published to npm with this name. I think it will eventually happen. I'm gonna hold off until I can add a few more features I would like. Then I'll change the name and move the npm module at the same time.
